context/db.go

```diff
@@ -21,6 +22,18 @@ import (
 
 var dbMutex sync.Mutex
 
+type UeStateForDB struct {
+	AmfUeBsonA  []byte
+	Supi        string
+	AmfUeNgapID int64
+	RanUeNgapID int64
+	GnbId       string
+	Guti        string
+}
+
+var StoreAmContextDbChannel chan UeStateForDB
+var DeleteAmContextDbChannel chan UeStateForDB
```

Comment:
This will fix the other `golangci-lint` lint issue
```suggestion
var (
       StoreAmContextDbChannel  chan UeStateForDB
       DeleteAmContextDbChannel chan UeStateForDB
)
```